mesh/options.go

```diff
@@ -0,0 +1,165 @@
+package mesh
+
+// Options.go holds the option settings for a single graph request.
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"k8s.io/client-go/tools/clientcmd/api"
+
+	"github.com/kiali/kiali/business"
+	"github.com/kiali/kiali/business/authentication"
+)
+
+// The supported vendors
+const (
+	VendorCytoscape     string = "cytoscape"
+	defaultConfigVendor string = VendorCytoscape
```

Comment:
Mesh is only implemented with Patternfly, right? Why the `cytoscape` reference?

Reply:
You are right, but this is sort of a legacy that can be cleaned up if/when we remove Cytoscape.  The fact is that even for the Patternfly graph, the JSON we return to the client is in Cytoscape format.  Meaning, the JSON can be passed directly into Cytoscape to form the graph.  But for Patternfly, we take the same cytoscape-compatible JSON and just parse it ourselves to define the PFT elements.  So, in effect, the config vendor actually is Cytoscape, for both graph renderings.